lib/inject_method.rb

```diff
@@ -0,0 +1,9 @@
+class Array
+  def owen_inject(&block)
```

Comment:
Unused method argument - `block`. If it's necessary, use `_` or `_block` as an argument name to indicate that it won't be used. You can also write as `owen_inject(*)` if you want the method to accept any arguments but don't care about them.